tests/orm/implementation/test_utils.py

```diff
@@ -20,24 +20,32 @@
 class TestOrmImplementationUtils:
     """Test the utility methods in aiida.orm.implementation.utils"""
 
-    def test_invalid_attribute_extra_key(self):
+    @pytest.mark.parametrize(
```

Comment:
Wow, `yapf` really went to town on this one. 😅 Let's see if I can fix that.

Reply:
Not sure how to get `yapf` to accept anything reasonable. For reference, this is what `black` makes of it:

```
    @pytest.mark.parametrize(
        "invalid_key",
        (
            5,
            f"invalid{FIELD_SEPARATOR}key",
            {"a": {5: "b"}},
            {"a": {f"invalid{FIELD_SEPARATOR}key": "b"}},
        ),
    )
```

And I get cake after the formatting too! 🙃